ddl/ddl_worker.go

```diff
@@ -214,10 +220,12 @@ func (d *ddl) handleDDLJobQueue() error {
 			schemaVer = d.runDDLJob(t, job)
 			if job.IsFinished() {
 				binloginfo.SetDDLBinlog(d.workerVars.BinlogClient, txn, job.ID, job.Query)
```

Comment:
Would you use the information in Job and will the `Cancel` status affect you?  @GregoryIan